src/views/GovernanceReviewTeam/index.scss

```diff
@@ -0,0 +1,29 @@
+.easi-grt {
+  &__request-summary {
+    background-color: #0071bc;
+    color: #fff;
```

Comment:
Could we use some of the utility classes for text color instead of defining our own CSS? https://designsystem.digital.gov/utilities/color/

Reply:
Because the `background-color` doesn't have a utility class, I didn't want to split out `color` into a class name. It might be confusing to have part of the styles live in HTML and part of them live in CSS.